spec/item_repository_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'item_repository'
+
+RSpec.describe ItemRepository do
```

Comment:
Good. Tests have good cover the methods for both item_repository_spec and order_repository_spec